beacon-chain/powchain/service.go

```diff
@@ -377,20 +377,38 @@ func (s *Service) dialETH1Nodes() (*ethclient.Client, *gethRPC.Client, error) {
 		return nil, nil, err
 	}
 	httpClient := ethclient.NewClient(httpRPCClient)
-
+	// Add a method to clean-up and close clients in the event
```

Comment:
test?

Reply:
Its a bit tricky here, as there is no way to simulate the rpc connection here. We could create a mock for this, but then that doesn't test much, just that the mock runs fine.

Reply:
understood, will approve